typesafety/test_issue_49.yml

```diff
@@ -7,3 +7,35 @@
     logger.debug(123.3)
     logger.trace(1)
     logger.error(.3)
+- case: variables
+  main: |
+    from loguru import logger
+
+    foo = "bar"
+    logger.info(foo)
```

Comment:
You can use `parametrized` and keep severity level as param making this test a lot shorter. There are examples how to do that in other cases :)